astronomy-pipeline/astronomy_lambda.py

```diff
@@ -0,0 +1,22 @@
+"""Full ETL pipeline lambda for the astronomy API data."""
+
+import logging
+
+from astronomy_extract import extract_weekly_astronomy_data
+from astronomy_transform import transform_astronomy_data
+from astronomy_load import upload_astronomy_data
+
+
+def lambda_handler(event=None, context=None) -> None:
+    '''Runs the notification pipeline'''
```

Comment:
Not to be too nitpicky, but this doc-string refers to the 'notification pipeline' which would be the hourly Lambda checking for visible bodies and events, while this is the weekly ETL that loads the celestial bodies into the RDS. It might be worth changing the docstring to reflect that a bit more closely.